src/box/lua/config/instance_config.lua

```diff
@@ -2420,6 +2420,12 @@ return schema.new('instance_config', schema.record({
         }, {
             default = 'old',
         }),
+        box_consider_system_spaces_synchronous = schema.enum({
+            'old',
+            'new',
+        }, {
+            default = 'old',
```

Comment:
Could you remind me, please, when do we set these compat options to `new` by default?

Reply:
AFAIC, on major releases.